Classes/BEMCheckBox.m

```diff
@@ -9,6 +9,7 @@
 #import "BEMCheckBox.h"
 #import "BEMAnimationManager.h"
 #import "BEMPathManager.h"
+#import "BEMCheckBoxGroup.h"
```

Comment:
If the `@class` is replaced with an `#import` statement in the header file as suggested with my previous comment, then this `#import` statement becomes unnecessary.